style/non-code/README.md

```diff
@@ -0,0 +1,19 @@
+# Non-code
+
+A spot to record non-code things, e.g. email signatures!
+
+## Email Signature
+
+```
+Jeremy Loyd
+Sparkbox, Creative Director
+123 Webster Street, Studio 2
+Dayton, OH 45402
+937.401.0915
+www.seesparkbox.com
+```
+
+- First line - `Arial, Bold, 16px, #33717a`
```

Comment:
ping @jeremyloyd for guidance....lead us! :smile: 
![getinsertpic.com](http://media3.giphy.com/media/TRPNcJMNJGyQ0/200.gif)